jaxrs-spec/src/main/asciidoc/chapters/appendix/_changes-since-3.0-release.adoc

```diff
@@ -0,0 +1,17 @@
+////
+*******************************************************************
+* Copyright (c) 2020 Eclipse Foundation
+*
+* This specification document is made available under the terms
+* of the Eclipse Foundation Specification License v1.0, which is
+* available at https://www.eclipse.org/legal/efsl.php.
+*******************************************************************
+////
+
+[[changes-since-3.0-release]]
+=== Changes Since 3.0 Release
+
+* <<exceptionmapper>>: Added requirement that JAX-RS implementations have 
+default exception mappers.
+* <<resource_field>>: Array types may be specified for `@CookieParam`,
+`@FormParam`, `@HeaderParam`, `@MatrixParam` and `@QueryParam` parameters.
```

Comment:
We should also include other changes here - including:
* Java SE APIs
* SseBroadcaster close(boolean cascading) method and behavior
* New `hasProperty` method to various classes/interfaces
* SameSite cookie support
* Support for arrays as Params
* Default exception mapper

Reply:
Note that this chapter does not list all changes of Jakarta REST but only the changes of the spec document. As these features are not yet covered by the spec document so far, it would be incorrect to list them here upfront. We should take care to correctly cover each new feature by API, JavaDoc, TCK, Spec doc, and changes doc, but these changes should go in separate PRs per feature.